docker/Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04 AS armory
 
 RUN apt-get -y -qq update && \
-    apt-get install -y wget vim build-essential git curl
+    apt-get install -y wget vim build-essential git
```

Comment:
Since this is the first step in the build of our base image it'll invalidate a lot of layer re-use. I think it's okay to keep curl in the container

Reply:
Updated.